app/src/main/java/com/hover/stax/data/local/channels/ChannelDao.kt

```diff
@@ -28,6 +29,9 @@ interface ChannelDao {
     @Query("SELECT * FROM channels WHERE country_alpha2 = :countryCode AND id IN (:channel_ids) ORDER BY name ASC")
     fun getChannels(countryCode: String, channel_ids: IntArray): LiveData<List<Channel>>
 
+    @Query("SELECT * FROM channels WHERE country_alpha2 = :countryCode AND id IN (:channel_ids) ORDER BY name ASC")
+    fun getChannelsByCountryCode(channel_ids: IntArray, countryCode: String): List<Channel>
```

Comment:
This method is the same as the one above except the return type. The argument order should be the same and the names should be similar but make clear how they are different.